sys/sys/uio.h

```diff
@@ -50,15 +50,28 @@ typedef	__off_t	off_t;
 #endif
 
 #ifdef _KERNEL
+#define	UIO_INLINE_IOV	2
```

Comment:
I think this deserves some comment explaining the choice, and maybe a hint on how to check whether it might be a suboptimal choice (check `vmstat -m | grep iov`).

Reply:
Added a comment here.

Reply:
Thanks. I think we could be a bit more precise: one should compare the number of M_IOV allocations with the number of allocations from the UMA zone. The ratio should be low, of course. Both numbers can be obtained from `vmstat`, dtrace is mostly useful for finding a good value for `UIO_INLINE_IOV`. The precise invocation I used is `dtrace -n 'dtmalloc::iov:malloc {@ = lquantize(args[3], 16, 4096, 8);}'`.